providers/implementations/storemgmt/file_store.c

```diff
@@ -481,10 +482,12 @@ static int file_set_ctx_params(void *loaderctx, const OSSL_PARAM params[])
         if (!OSSL_PARAM_get_octet_string_ptr(p, (const void **)&der, &der_len)
             || (x509_name = d2i_X509_NAME(NULL, &der, der_len)) == NULL)
             return 0;
-        hash = X509_NAME_hash(x509_name);
+        hash = X509_NAME_hash_ex(x509_name, NULL, NULL, &ok);
```

Comment:
Please use `ossl_prov_ctx_get0_libctx(ctx->provctx)` for the libctx. Keep propq at NULL.

Reply:
Good point, done.